gremlin/values.yaml

```diff
@@ -292,6 +295,15 @@ chao:
 
   podLabels: {}
 
+  serviceAccount:
+    # Determines if the `chao` deployment should be installed. This deployment launches an agent that enabled Kubernetes
+    #   targeting for Gremlin attacks
+    create: true
+    # Cloud IAM role to use to resolve any remote secrets
+    iamRole:
```

Comment:
I think we can remove this now that we're exposing this generically via `annotations`, that wait we're kubernetes-platform agnostic for now.

Reply:
Yeah makes sense